wrappers/src/error_handling.cpp

```diff
@@ -32,6 +32,13 @@
 
 namespace realm {
 
+    SetDuplicatePrimaryKeyValueException::SetDuplicatePrimaryKeyValueException(std::string object_type, std::string property, std::string value)
+        : std::runtime_error(util::format(
+            "Primary key property '%1.%2' already has a primary key '%3'",
```

Comment:
This kindof sounds like the property was already set to said value. Perhaps `"A %1 object already exists with primary key %2 == '%3'"` or something similar?